### ISSUE TITLE
Version 2 references to v1, is it okay?

### ISSUE THREAD

**Issue by d1bevz** [hvac/hvac#824]
https://github.com/hvac/hvac/blob/e84c959989568f9dbac494a7b4215116555a98e6/hvac/api/secrets_engines/kv_v2.py#L67

**Comment by colin-pm** [comment]
Yes, the v1 is still expected for the KV version 2 API.  Please refer to the docs for URL examples:
https://www.vaultproject.io/api-docs/secret/kv/kv-v2#sample-request-1
